workflow-beta1/manifests/deis-minio-rc.yaml

```diff
@@ -17,8 +17,8 @@ spec:
       serviceAccount: deis-minio
       containers:
         - name: deis-minio
-          image: quay.io/deis/minio:v2.0.0-beta1
-          imagePullPolicy: IfNotPresent
+          image: quay.io/{{.minio.org}}/minio:{{.minio.docker-tag}}
```

Comment:
Did you test this because last when i did helm template failed when it saw a '-' character